sources/packages/backend/libs/integrations/src/esdc-integration/student-loan-balances/student-loan-balances.integration.service.ts

```diff
@@ -34,7 +40,10 @@ export class StudentLoanBalancesIntegrationService extends SFTPIntegrationBase<S
         `The Student loan balances file ${remoteFilePath} has an invalid record type on header: ${header.recordTypeCode}.`,
       );
       // If the header is not the expected one, throw an error.
-      throw new Error("Invalid file header.");
+      throw new CustomNamedError(
+        "Invalid file header.",
+        STUDENT_LOAN_BALANCE_INVALID_FILE_HEADER,
```

Comment:
Do we really need the custom named error here?? Because we are not doing any translation or logic based on the custom named error, I think we don't need it.